quadfeather/tiler.py

```diff
@@ -454,15 +458,33 @@ def __init__(
         self._macrotiles: Optional[List[Macrotile]] = None
         self._bloom_cache: Dict[str, pa.Array] = {}
 
+    # Easier file system handling methods
+
+    def write_feather(self, write_path: Path, table: pa.Table, compression: Literal["zstd", "uncompressed"] = "zstd"):
+        with self.filesystem.open_output_stream(write_path.as_posix()) as f:
+            feather.write_feather(table, f, compression=compression)
+
+    def read_feather(self, read_path: Path, columns: Optional[List[str]] = None) -> pa.Table:
+        with self.filesystem.open_input_file(read_path.as_posix()) as f:
+            return feather.read_table(f, columns=columns)
+
+    def file_exists(self, path: Path) -> bool:
+        return self.filesystem.get_file_info(path.as_posix()).is_file
+
     @staticmethod
-    def from_dir(basedir: Path, mode: Literal["read", "append"]) -> "Quadtree":
+    def from_dir(basedir: Path, mode: Literal["read", "append"], filesystem: Optional[fs.FileSystem] = None) -> "Quadtree":
         # Load a quadtree from disk.
-        manifest = basedir / "manifest.feather"
-        if not manifest.exists():
+        if filesystem is None:
+            filesystem = fs.LocalFileSystem()
```

Comment:
```suggestion
```

Reply:
Probably we don't need to do all this, but I think the `Optional`s above just made it possible for people to override the local filesystem default by specifically passing None, which would then use the local filesystem default.